### PR TITLE
fixing navigation

### PR DESCRIPTION
**Pull request by vesper2000**
#minor#

### CODE REVIEW

File: addon/components/frost-application-bar.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-application-bar'
```
Comment (rox163) [inline addon/components/frost-application-bar.js]: app bar is part of navigation? The nav component was only supposed to be the modal dialog to allow for it to be launched from any kind of component a consumer may want

Comment (vesper2000) [inline addon/components/frost-application-bar.js]: It is just a new wrapper that we use, nothing to do with the actual ember-frost-app-bar repo.

Comment (rox163) [inline addon/components/frost-application-bar.js]: ah k. the name threw me off
👍

**Comment by rox163** [comment]
Approved

**Pull request merged**